config/locales/de.yml

```diff
@@ -225,10 +227,14 @@ de:
   telegram:
     unknown_content_message: |
       Unsere Software kann derzeit Textnachrichten und Bilder verarbeiten, jedoch noch keine weiteren Inhalte wie z.B. Sprachnachrichten oder Dateianhänge. Für die Zwischenzeit bitten wir dich, Nachrichten mit solchen Inhalten an @FrauCsu (Astrid Csuraji) zu schicken. Vielen Dank!
+
+      Ohjelmistomme pystyy tällä hetkellä käsittelemään teksti- ja ääniviestejä ja kuvia, mutta ei muita sisältöjä, kuten esim. tiedostoliitteitä. Voisitteko ystävällisesti lähettää viestit jossa on liite osoitteeseen @FrauCsu (Astrid Csuraji). Paljon kiitoksia!
     welcome_message: |
-      Herzlich Willkommen bei %{project_name}. Danke, dass Sie an unserer Dialog-Recherche teilnehmen. In den kommenden Wochen nutzen wir Telegram für den Austausch. Ein Bot arbeitet wie ein Postbote für uns und stellt Fragen und Antworten zu. Ihre Nachrichten lesen und beantworten wir aber selbstverständlich persönlich. Wir freuen uns sehr auf den Dialog mit Ihnen!
+      Herzlich Willkommen beim #Schweinesystem. Danke, dass Sie an unserer Dialog-Recherche teilnehmen. In den kommenden Wochen nutzen wir Telegram für den Austausch. Ein Bot arbeitet wie ein Postbote für uns und stellt Fragen und Antworten zu. Ihre Nachrichten lesen und beantworten wir aber selbstverständlich persönlich. Wir freuen uns sehr auf den Dialog mit Ihnen!
```

Comment:
@roschaefer I removed the ability to set the project name via env variables for this translation, as it doesn’t work for this particular multi-lang project. I guess that’s okay, as it’s soon going to be outdated anyways when we merge #434?

Reply:
That's OK

Curious: why does it not work?

Reply:
It doesn’t work was maybe not accurate. The problem is that we need to translate the project name as well in this case.